Trophy.md

```diff
@@ -0,0 +1,31 @@
+# 🏆 Bugs & CVEs 
+
+The showcase of bugs and CVEs found by *UTopia*-based fuzzers serves a variety of purpose:
+- Show the effect of the fuzzing automation approach (i.e.,*UTopia* can help find more bugs in the projects that are already registered and have been being fuzz tested in OSS-Fuzz)
+- Provide insight into making unit tests that can work well with *UTopia* (i.e., Fuzzing friendly unit tests)
+- Increase the visibility of various fuzz drivers so that people can learn or improve fuzz driver composing strategies
+
+Note that The merged fuzz drivers are different from the original ones generated by *UTofia* because they are reviewed and modified by the community.
+
+Project | Fuzzer | Bugs or CVEs | Category 
+----- | ----------- | ------ | ---------
+[OpenCV](https://github.com/opencv/opencv) | [readnetfromtensorflow_fuzzer.cc](https://github.com/google/oss-fuzz/pull/7781) | [#21947](https://github.com/opencv/opencv/issues/21947) | `bof`
+
+
+
+
+### Description of categories:
+
+* `arith`: Arithmetic error (e.g., overflows, divide-by-zero)

```

Comment:
Checks. We may not cover all categories, since UTopia generated fuzzers will find bugs based on address sanitizer.

Reply:
You're right, it's not a complete or necessary category. Let's start with this incomplete category and polish them further. I removed